ocaml/testsuite/tests/tool-ocamlobjinfo/question.ml

```diff
@@ -8,6 +8,11 @@ program = "question.cmxs"
 **** check-ocamlopt.byte-output
 ***** ocamlobjinfo
 ****** check-program-output
+
+***** ocamlobjinfo
+program = "question.cmx"
+(* The cmx output varies too much to check. We're just happy it didn't
+   segfault on us. *)
```

Comment:
I'd like a test where there is a function in the code, as the cmx file corresponding to this code does not contain a function, and thus does not contain code sections.